README.md

```diff
@@ -39,6 +39,12 @@ In your browser, go to
 
 - `http://localhost:3000`
 
+##Live Demo
+
+-[Live Demo](https://git.heroku.com/young-harbor-90243.git)
+
+-[Video Presentation](https://www.loom.com/share/151a19a3af134ace9a401ad252dc9272)
+
```

Comment:
- Good presentation, but you can always improve on it. I'd suggest you do a quick work-through of the UI(user interface) and more time working your audience through your code, telling them what you are most proud of about your code and what was done, it shows you know what you are doing 👍🏽.

Reply:
- Kindly make sure your UI matches that of the wireframe given. 👍🏽

Reply:
- Lastly, make sure you merge your branch with the develop branch and request for review from there 👍🏽.